Source/JavaScriptCore/interpreter/Interpreter.cpp

```diff
@@ -122,11 +122,17 @@ JSValue eval(CallFrame* callFrame, JSValue thisValue, JSScope* callerScopeChain,
         return jsUndefined();
 
     JSValue program = callFrame->argument(0);
-    if (!program.isString())
+    JSString* programString = nullptr;
```

Comment:
This can't result in a behaviour difference with the flag disabled because codeForEval only ever returns a string from a TrustedScript object that you wouldn't be able to create.

Having said that any pointers on how to access the feature flag from within JSC would be appreciated.

Reply:
You can add a new option to OptionsList.h